tests/test_arithmetics.rs

```diff
@@ -423,3 +423,193 @@ fn rich_comparisons_python_3_type_error() {
     py_expect_exception!(py, c2, "c2 >= 1", PyTypeError);
     py_expect_exception!(py, c2, "1 >= c2", PyTypeError);
 }
+
+#[cfg(not(feature = "no-return-notimplemented"))]
+#[cfg(test)]
+mod not_implemented {
+    use super::*;
+
+    #[pyclass]
+    struct RichComparisonToSelf {}
+
+    #[pyproto]
+    impl<'p> PyObjectProtocol<'p> for RichComparisonToSelf {
+        fn __repr__(&self) -> &'static str {
+            "RC_Self"
+        }
+
+        fn __richcmp__(&self, other: PyRef<'p, Self>, _op: CompareOp) -> PyObject {
+            other.py().None()
+        }
+    }
+
+    #[pyproto]
+    impl<'p> PyNumberProtocol<'p> for RichComparisonToSelf {
```

Comment:
I think there are too many tests.
Two or three operators are sufficient.

Reply:
I've just push commit 5a0cdbe1f78f23c8189be1832b6506a44156737b.

I kept all the operators for two reasons:

- It helps to have a list comprehensive list of which operators implement.

- Although right now all the operators are implemented nicely in a macro, having them here can help to detect errors in future refactors.

Reply:
However, I group them in just four tests.

Reply:
Reworked the tests in commit 366892fb2bdd96e7b5ad770464907d276e974222.